types/test_utils/mocks/account.go

```diff
@@ -0,0 +1,24 @@
+package mocks
```

Comment:
The mocks we use the most are in this package. The functions simply return the value that the structure has.